cmd/resources/resources.go

```diff
@@ -137,7 +137,7 @@ func NewOperationCmd(parentCmd *cobra.Command, client *http.Client, op Operation
 	}
 
 	opCmd.cmd = cmd
-	opCmd.initFlags()
+	_ = opCmd.initFlags()
```

Comment:
Fixes lint error `Error return value of `opCmd.initFlags` is not checked (errcheck)`